### ISSUE TITLE
Improve styles for invalid form

### ISSUE THREAD

**Issue by zipymonkey** [psu-libraries/psulib_base#256]
Forms using HTML Client Validation are not getting any of the bootstrap error styles.  

The simplest way to opt out is to add the `novalidate` attribute to your <form>. You can also set `formnovalidate` on your submit controls. 

## Acceptance Criteria
- [ ] Use Bootstrap form validation styles fro invalid fields (`:invalid`).  We probably need to do this with some sort of mixing link the 
- [x] Disable smooth scrolling on webform nodes and webforms (HTML Client Side Validation pop ups are not displaying in smooth scrolling is enabled and the form input needs to scroll into view.)
- [ ] Investigate if we need to keep the default bootstrap validation (or if it should be swapped out).

## Links
- https://developer.mozilla.org/en-US/docs/Learn_web_development/Extensions/Forms/Form_validation
- https://getbootstrap.com/docs/5.3/forms/validation/
- https://mounir.lamouri.fr/2010/11/html5-forms-validation-in-firefox-4.html
- https://issues.chromium.org/issues/40196176 and https://stackoverflow.com/a/77222946
